test/diff-test.html

```diff
@@ -85,6 +85,9 @@ <h1>${heading}</h1>
         <p>${heading}</p>
         <a href="/foo/${heading}"></a>
         <slot></slot>
+        <script>
+          console.log("Script should be removed")
+        <${'/script>'}
```

Comment:
Is there some reason this can't be `</script>`?

Reply:
Yes, if it is just a closing script tag it will prematurely close the surrounding script tag for the test setup. That was just the easiest way I could see to avoid that.